privacyidea/lib/crypto.py

```diff
@@ -190,6 +188,8 @@ def pass_hash(password):
     :type password: str
     :return: The hash string of the password
     """
+    pass_ctx = CryptContext(get_app_config_value("PI_HASH_ALGO_LIST",
```

Comment:
Or we should add the `argon2__rounds` here, so that they are set.

The question however is, what would happen, if we provide the keyword parameter, but Argon2 is not contained in the list!

Reply:
fixed in 19fb6e0d342f1b141fcfb10a784ab213afda6847.

Reply:
fixed in 19fb6e0d342f1b141fcfb10a784ab213afda6847.

Reply:
Adding the `argon2__` parameter doesn't break even if `argon2` is not in the list of hash algorithms.